src/web/modules/webhooks/webhooks.http.ts

```diff
@@ -15,7 +15,8 @@ export async function list(req: Request, res: Response, next: NextFunction): Pro
     const serviceId = req.params.serviceId
 
     const response = await Webhooks.webhooks.list({
-      service_id: serviceId as string
+      service_id: serviceId as string,
+      live: true
```

Comment:
I know this page isn't in use yet but I think it would be good to avoid hardcoding values for now (for example its generally our test services that have webhooks configured at the moment). I think it would be good for this to work as closely as possible with our existing "list" pages, we can then decide later if we want to change how they work all together.

I think to achieve that here we can make some small changes: 

The [URL](https://github.com/alphagov/pay-toolbox/blob/master/src/web/router.js#L164) would just be `/webhooks` 

The endpoint would accept a query parameter called `account`.

This query parameter can then be set when linking the gateway account page with this page. (example `/webhooks?account=123`)

When this controller loads, it will have all of the information needed to make the request to the webhooks service, with something like: 

```js
const accountId = req.query.account

if (!accountId) {
  throw new Error('Webhooks can currently only be shown for a single account')
}

const account = await Connector.accounts.retrieveAPI(accountId)
const service = await AdminUsers.services.retrieve(account.service_id)
const { results } = await Webhooks.webhooks.list({
  service_id: acccount.service_id,
  live: account.type === AccountType.LIVE 
})
```

Reply:
Thanks for this - I will update it